peps/pep-0782.rst

```diff
@@ -0,0 +1,371 @@
+PEP: 782
+Title: Add PyBytesWriter C API
+Author: Victor Stinner <vstinner@python.org>
+Discussions-To: https://discuss.python.org/t/81182
```

Comment:
Discussions-To should be updated with a new thread once this PEP is merged, for now we can move this link to Post-History.
```suggestion
```

Reply:
Ok, will do.